kie-drools-wb-parent/kie-drools-wb-webapp/src/main/resources/org/kie/workbench/drools/KIEDroolsWebapp.gwt.xml

```diff
@@ -72,6 +72,7 @@
   <inherits name="org.kie.workbench.common.services.KieWorkbenchCommonServicesAPI"/>
   <inherits name='org.kie.workbench.common.services.datamodel.KieWorkbenchCommonDataModelAPI'/>
   <inherits name='org.kie.workbench.common.services.refactoring.KieWorkbenchCommonRefactoringAPI'/>
```

Comment:
Same comment as for the 7.2.x PR.. did you want to include the inherit for ```API```? (You remove it for ```kie-wb```).

It does no harm (assuming ```Client``` inherits ```API```) however it's nice to keep the dependency lists in these two webapps consistent where applicable so things are clearer (less ambiguity) for people looking in the future.